CHANGES.md

```diff
@@ -1,5 +1,10 @@
 # Change Log
 
+
+## 0.11.1
```

Comment:
Would be better to use `0.12.0` version as this PR adds some new functionality, instead of being a bugfix.

Reply:
Hi, just changed the version in CHANGES.md and pack.yaml as suggested.